client/my-sites/site-settings/controller.js

```diff
@@ -64,8 +64,7 @@ module.exports = {
 	siteSettings( context ) {
 		let analyticsPageTitle = 'Site Settings';
 		const basePath = route.sectionify( context.path );
-		const fiveMinutes = 5 * 60 * 1000;
-		let site = sites.getSelectedSite();
+		const site = getSelectedSite( context.store.getState() );
```

Comment:
I'll revisit this more exhaustively because this error is happening in other components too. Look like I was not clearing the cache properly when testing this